c/parallel/src/nvrtc/ltoir_list_appender.h

```diff
@@ -15,24 +15,42 @@
 #include "command_list.h"
 #include <cccl/c/types.h>
 
-struct nvrtc_ltoir_list_appender
+struct nvrtc_linkable_list_appender
 {
-  nvrtc_ltoir_list& ltoir_list;
+  nvrtc_linkable_list& linkable_list;
 
-  void append(nvrtc_ltoir lto)
+  void append(nvrtc_linkable linkable)
   {
-    if (lto.ltsz)
+    std::visit(
+      [&](auto&& l) {
+        if (l.size)
+        {
+          linkable_list.push_back(std::move(l));
+        }
+      },
+      linkable);
```

Comment:
This maybe a naive question, but I was not sure why `std::move` is safe here. `std::visit` takes the `linkable` argument as rvalue-reference. It is perhaps being copied when `append` call is made.